packages/components/src/components/hds/table/index.ts

```diff
@@ -87,13 +87,19 @@ export interface HdsTableSignature {
 }
 
 export default class HdsTable extends Component<HdsTableSignature> {
-  @tracked sortBy = this.args.sortBy ?? undefined;
-  @tracked sortOrder = this.args.sortOrder || HdsTableThSortOrderValues.Asc;
+  @tracked sortBy;
+  @tracked sortOrder;
   @tracked selectAllCheckbox?: HdsFormCheckboxBaseSignature['Element'] =
     undefined;
   selectableRows: HdsTableSelectableRow[] = [];
   @tracked isSelectAllCheckboxSelected?: boolean = undefined;
 
+  constructor(owner: unknown, args: HdsTableSignature['Args']) {
+    super(owner, args);
+    this.sortBy = this.args.sortBy ?? undefined;
+    this.sortOrder = this.args.sortOrder || HdsTableThSortOrderValues.Asc;
```

Comment:
suggestion, non-blocker
```suggestion
    this.sortOrder = this.args.sortOrder ?? HdsTableThSortOrderValues.Asc;
```